lib/logitech_receiver/hidpp20_constants.py

```diff
@@ -181,17 +181,18 @@ class FeatureFlag(IntFlag):
 
 CHARGE_TYPE = NamedInts(standard=0x00, fast=0x01, slow=0x02)
 
-ERROR = NamedInts(
-    unknown=0x01,
-    invalid_argument=0x02,
-    out_of_range=0x03,
-    hardware_error=0x04,
-    logitech_internal=0x05,
-    invalid_feature_index=0x06,
-    invalid_function=0x07,
-    busy=0x08,
-    unsupported=0x09,
-)
+
+class Error(IntEnum):
+    unknown = (0x01,)
```

Comment:
I will abandon this part, since it conflicts with https://github.com/pwr-Solaar/Solaar/pull/2647. I am waiting for its merge, then aply the logic on hidpp10_constants